lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -296,7 +303,7 @@ sub _generate_jobs {
             push @parents, _parse_dep_variable($ret->[$i]->{START_AFTER_TEST}, $ret->[$i]);
             push @parents, _parse_dep_variable($ret->[$i]->{PARALLEL_WITH},    $ret->[$i]);
             for my $p (@parents) {
-                $wanted{$p} = 1;
+                $wanted{join(':', @$p)} = 1;
```

Comment:
Am I the only one who prefers `$parent` or even `$parent_name` over just `$p`?

Reply:
Thanks for review, fixed